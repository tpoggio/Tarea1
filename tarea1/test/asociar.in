asociar 15 5.2
esClave 0
esClave 15
asociar 15 5.2
esClave 15
Fin