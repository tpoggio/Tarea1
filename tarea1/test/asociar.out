1>Asociar.
2>0 no es clave. 
3>15 es clave. 
4>Asociar.
5>15 es clave. 
6>Fin.
